backend/.github.backup/workflows/test.yml

```
@@ -0,0 +1,269 @@
+        cache-from: type=gha
+        cache-to: type=gha,mode=max
+        
+    - name: 🧪 Test Docker Container
+      run: |
+        docker run --name test-container -d \
+          -e DATABASE_URL="postgresql://test:test@host.docker.internal:5432/test" \
+          -e JWT_SECRET="test-secret" \
+          -e JWT_REFRESH_SECRET="test-refresh-secret" \
+          -p 3000:3000 \
+          task-management-backend:test
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)